daprdocs/content/en/reference/components-reference/supported-bindings/s3.md

```diff
@@ -65,6 +65,7 @@ The above example uses secrets as plain strings. It is recommended to use a secr
 | `encodeBase64` | N | Output | Configuration to encode base64 file content before return the content. (In case of opening a file with binary content). `"true"` is the only allowed positive value. Other positive variations like `"True", "1"` are not acceptable. Defaults to `"false"` | `"true"`, `"false"` |
 | `disableSSL`     | N        | Output | Allows to connect to non `https://` endpoints. Defaults to `"false"` | `"true"`, `"false"` |
 | `insecureSSL`     | N        | Output | When connecting to `https://` endpoints, accepts invalid or self-signed certificates. Defaults to `"false"` | `"true"`, `"false"` |
+| `storageClass` | N | Output | The desired storage class for objects during the create operation | `STANDARD_IA` |
```

Comment:
Are there other options than just STANDARD here? I am sure there are and it would be good to look at the code to see what others are possible

Reply:
You also need to add this into the Yaml example above.

Reply:
looks like there are more options based [on here,](https://aws.amazon.com/s3/storage-classes/) but in components contrib I can only see `STANDARD_IA` as an example of the field being filled in.

Reply:
For the Yaml, it needs to be added to here https://docs.dapr.io/reference/components-reference/supported-bindings/s3/#component-format

Reply:
> looks like there are more options based [on here,](https://aws.amazon.com/s3/storage-classes/) but in components contrib I can only see `STANDARD_IA` as an example of the field being filled in.

We should clarify if any of the storage classes can be used. I presume yes, which if the case we can put a link to that AWS document.

Reply:
I would presume yes as well, that any valid storageClass type can be passed in. Let me find some docs to link to